__tests__/endpoints.test.js

```diff
@@ -91,3 +91,43 @@ describe("GET /api/articles/:article_id", () => {
     })
 })
 
+describe("/api/articles", () => {
+    test("200: returns object with all the articles", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.body
+            
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                })
+            })
+        })
+    })
+    test("articles appear in descending order", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.body
+            expect(articles[0].article_id).toBe(articles.length)
+        })
+    })
+    test("when given a bad query endpoint returns Bad Request error", () => {
+        return request(app)
+        .get("/api/articles?banana=banana")
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    })
```

Comment:
Im not sure what this test is checking for exactly, but at this point we shouldnt be expecting any user input so there is no valid error handling that needs to be added for this endpoint yet.